api-spec/STAC.yaml

```diff
@@ -738,7 +738,7 @@ components:
           $ref: '#/components/schemas/time'
     intersectsFilter:
       type: object
-      description: Only returns items that intersect with the provided geometry.
+      description: Only returns items that intersect with the provided polygon.
```

Comment:
I just ran `npm run generate-all`, and one recent change hadn't been picked up into this file yet.